prow/scripts/cluster-integration/kyma-cli-alpha-uninstall.sh

```diff
@@ -0,0 +1,192 @@
+# Once we figure out a way to fix this, this custom CR can be deleted from this script.
+cat << EOF > "/tmp/kyma-parallel-install-installationCR.yaml"
+apiVersion: "installer.kyma-project.io/v1alpha1"
+kind: Installation
+metadata:
+  name: kyma-installation
+  namespace: default
+spec:
+  components:
+    - name: "testing"
+      namespace: "kyma-system"
+    - name: "knative-eventing"
+      namespace: "knative-eventing"
+    - name: "dex"
+      namespace: "kyma-system"
+    - name: "ory"
+      namespace: "kyma-system"
+    - name: "api-gateway"
+      namespace: "kyma-system"
+    - name: "rafter"
+      namespace: "kyma-system"
+    - name: "service-catalog"
+      namespace: "kyma-system"
+    - name: "service-catalog-addons"
+      namespace: "kyma-system"
+    - name: "helm-broker"
+      namespace: "kyma-system"
+    - name: "nats-streaming"
+      namespace: "natss"
+    - name: "core"
+      namespace: "kyma-system"
+    - name: "cluster-users"
+      namespace: "kyma-system"
+    - name: "logging"
+      namespace: "kyma-system"
+    - name: "permission-controller"
+      namespace: "kyma-system"
+    - name: "apiserver-proxy"
+      namespace: "kyma-system"
+    - name: "iam-kubeconfig-service"
+      namespace: "kyma-system"
+    - name: "serverless"
+      namespace: "kyma-system"
+    - name: "knative-provisioner-natss"
+      namespace: "knative-eventing"
+    - name: "event-sources"
+      namespace: "kyma-system"
+    - name: "application-connector"
+      namespace: "kyma-integration"
+    - name: "tracing"
+      namespace: "kyma-system"
+    - name: "monitoring"
+      namespace: "kyma-system"
+    - name: "kiali"
+      namespace: "kyma-system"
+    - name: "console"
+      namespace: "kyma-system"
+EOF
+
+(
+cd "${KYMA_PROJECT_DIR}/kyma"
+set -x
+kyma alpha deploy \
+    --ci \
+    --resources "${KYMA_PROJECT_DIR}/kyma/resources" \
+    --components "/tmp/kyma-parallel-install-installationCR.yaml"
+)
+
+sleep 1m
+
+log::info "Uninstall Kyma"
+(
+cd "${KYMA_PROJECT_DIR}/kyma"
+set -x
+kyma alpha uninstall \
+    --ci \
+    --components "/tmp/kyma-parallel-install-installationCR.yaml"
+)
+
+sleep 1m
+
+log::info "Install Kyma again"
+(
+cd "${KYMA_PROJECT_DIR}/kyma"
+set -x
+kyma alpha deploy \
+    --ci \
+    --resources "${KYMA_PROJECT_DIR}/kyma/resources" \
+    --components "/tmp/kyma-parallel-install-installationCR.yaml"
+)
+
+log::info "Run Kyma tests"
+(
+cd "${KYMA_PROJECT_DIR}/kyma"
+set -x
+kyma test run \
+    --name "testsuite-alpha-$(date '+%Y-%m-%d-%H-%M')" \
+    --concurrency 6 \
+    --max-retries 1 \
+    --timeout 60m \
+    --watch \
+    --non-interactive \
+    istio-kyma-validate application-connector application-operator application-registry \
+    connection-token-handler connector-service api-gateway apiserver-proxy cluster-users \
+    console-backend core-test-external-solution dex-connection dex-integration kiali \
+    logging monitoring rafter serverless serverless-long service-catalog
```

Comment:
Shouldn't we run all of the tests here?

Reply:
I couldn't find a better way to "Run all but one" test, as `console-web` tests are failing with this type of installation due to a known bug.

Reply:
Ok I think it can be left as it is. We need to think if we want to extend the functionality of `gardener::test_kyma` to allow passing to Kyma CLI which tests to run.